src/controllers/logbook.controller.ts

```diff
@@ -174,60 +168,16 @@ export class LogbookController {
           await this.utils.renewAccessToken();
           continue;
         } else {
+          if (err.statusCode === 401) {
+            throw new HttpErrors.Unauthorized(err.message);
+          }
           console.error(err);
         }
       }
       break;
     } while (true);
   }
 
-  @authenticate("jwt")
-  @post("/Logbooks", {
-    responses: {
-      "200": {
-        description: "Create Room Response",
-        content: {
-          "application/json": {
-            schema: {
-              type: "object",
-              properties: {
-                room_id: {
-                  type: "string",
-                },
-                room_alias: {
-                  type: "string",
-                },
-              },
-            },
-          },
-        },
-      },
-    },
-  })
-  async create(
-    @requestBody(createLogbookRequestBody) details: CreateLogbookDetails,
-  ): Promise<{ room_alias: string; room_id: string } | undefined> {
-    do {
-      try {
-        const { name, invites } = details;
-        return await this.utils.createRoom(name, invites);
-      } catch (err) {
-        if (
-          err.error &&
-          (err.error.errcode === "M_UNKNOWN_TOKEN" ||
-            err.error.errcode === "M_MISSING_TOKEN")
-        ) {
-          await this.utils.renewAccessToken();
-          continue;
-        } else {
-          console.error(err);
-        }
-      }
-      break;
-    } while (true);
-  }
-
-  @authenticate("jwt")
   @get("/Logbooks/{name}", {
```

Comment:
Users whomever are able to see logbook tab and able to click into it should be able to see logbook messages, we already have that permission logic somewhere in the scicat BE and scicat FE. So I figured that JWT auth step is redundant.
As loopback service will be running in a SSH protected cluster I don't see open URL would cause any harm to our side. 
But one alternative way I think we could do is save synapse token in the scicat BE after first login, then trigger synapse service using token received from header.